web/impact/impact/tests/api_test_case.py

```diff
@@ -129,7 +132,30 @@ def assert_notified(self,
                 self.assertTrue(any([
                     message in email.body for email in emails]))
         if subject:
-            self.assertIn(subject, [email.subject for email in emails])
+            self.assertIn(subject, [email for email in emails])
+
+    def assert_email_attachments(self,
+                                 user,
+                                 location="",
+                                 meeting_info="",
+                                 topics=""):
+        '''Assert that the email attachments are working as expected
+        '''
+        emails = [email for email in mail.outbox if user.email in email.to]
+        attachments = [email.attachments[0][1] for email in emails]
```

Comment:
The magic numbers 0 and 1 here are alarming. We're always going to use this assert function to check the second attachment of the first email in the outbox? What if we ever want to look at the first attachment of the second email?

Reply:
removed these assertions